src/lib/types/restrictions.ts

```diff
@@ -113,7 +113,7 @@ export const stepValidation = z.object({
   FixedInputMetadataValue: fixedInputMetadataValue.optional(),
   FixedOutputMetadataValue: fixedOutputMetadataValue.optional(),
   FixedOutputMetadataValueType: fixedOutputMetadataValueType.optional(),
-})
+}).strict()
```

Comment:
makes `Zod` throw an error for invalid keys

Reply:
I was stuck for ages on a `create` failing at the transaction point. Turned out it was because it had an incorrectly keyed restriction. Before `strict()` that incorrect key was getting ignored by `Zod` with no error

Reply:
with `strict()` it throws nice errors like this:
```
errors: [
    {
      code: 'unrecognized_keys',
      keys: [Array],
      path: [],
      message: "Unrecognized key(s) in object: 'bla'"
    }
  ]
 ```